tokio/src/macros/loom.rs

```diff
@@ -1,11 +1,7 @@
 macro_rules! if_loom {
     ($($t:tt)*) => {{
         #[cfg(loom)]
-        const LOOM: bool = true;
-        #[cfg(not(loom))]
-        const LOOM: bool = false;
-
-        if LOOM {
+        {
```

Comment:
Fixes CI on `i686-unknown-linux-gnu`

Reply:
...why were we doing it this way in the first place, anyway? 🤔